### PR TITLE
Add sinkhole

### PR DESCRIPTION
**Pull request by qoelet**

### CODE REVIEW

**Comment by qoelet** [comment]
@sebdeckers a quick proof of concept, needs to be refined further, can you see if I'm on track?

File: dohnut.js [hunk 1]
```diff
@@ -13,9 +15,25 @@ program
...
+  blocked =
+    fs.readFileSync(program.sink, 'utf-8')
+      .split('\n')
+      .filter(Boolean)
+      .reduce(function(acc, cur, i) {
+        acc[cur] = 1
+        return acc
+      }, {})
+} catch (err) {
+  console.error('Unable to read given blocklist %j', program.sink)
```
Comment (sebdeckers) [inline dohnut.js:33]: Wow that string substitution syntax works? TIL!

File: dohnut.js [hunk 2]
```diff
@@ -25,6 +43,23 @@ const { origin, pathname } = new URL(doh)
 const client = http2.connect(origin)
 const socket = dgram.createSocket('udp4')
 
+const dumb = (id, rec) => {
+  return dnsPacket.encode({
+    type: 'response',
```
Comment (sebdeckers) [inline dohnut.js:48]: https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml

https://github.com/mafintosh/dns-packet/blob/0f87ff09f523447165848086534c477ca51337fd/rcodes.js

Refused? NXdomain?

Comment (qoelet) [reply dohnut.js:48]: I opt for lying and replying `NXDOMAIN`

Comment (sebdeckers) [reply dohnut.js:48]: SGTM

**Comment by sebdeckers** [comment]
So Good. Thanks, Mate!

**Pull request merged**